### PR TITLE
test: wanna remove the vendor file doesn't exist

### PR DESCRIPTION
**Pull request by yorkie**
The test/vendor/runner.js doesn't exist in current codebase, so it
causes failure of `duplo test`. So this commit is going to remove that
`cat` lines.

@kenhkan i still have a long time that don't touch the codebase of duplo, so i'm unsure if the location: `etc/test/vendor/runner.js` is imported by me, do you remember when do you remove this file? Thanks, i will still trace something about the issue, though :)

### CODE REVIEW

**Comment by yorkie** [comment]
And I added a flag `--no-runner` when running `duplo test`, so the complete usage would be like the following line:

```
$ sudo duplo test --no-runner
```

The reason to add this flag is for pixbi/indesign-plugin, because the environment of that is complicated a bit, so we could not make use of the jscoverage and mocha-phantomjs to automate the testing progress.

Such that I did implement the test suite for indesign-plugin by only building installer with tests, and install it in real indesign env, in the end, when we launch our pixbi plugin, the test suite would start running, so we would get result after few seconds, yea, currently we could not bind the test with CircleCI to automate this process, so that means we should run the test before every release.

@kenhkan do you have a better solution to this issue? and could you take a review at this :)

**Comment by kenhkan** [comment]
I actually haven't touched that file or any reference to it. If t's best to remove the file, feel free to remove the file directly in this PR.

**Comment by kenhkan** [comment]
Oh sorry. GitHub somehow didn't show your second comment. That sounds good. If `--no-runner` is sufficient, all is well. I suspect that we may need a `--dps` because there would be edge cases, but I'll let you tell me. :)

File: util/run-test.sh [hunk 1]
```diff
@@ -34,22 +35,53 @@ show_coverage_summary () {
 }
 
 # TODO: refactor and move this to `Scripts.hs` when proven to work
```
Comment (kenhkan) [inline util/run-test.sh]: @yorkie You're still in the process of adjusting the test framework for DPS, right?

Comment (yorkie) [inline util/run-test.sh]: Yea, will add tag "ready for merge" once i have done :)

**Comment by yorkie** [comment]
@kenhkan actually for now, the `--dps` would be equal to `--no-runner`, but let me add it.

**Comment by yorkie** [comment]
@kenhkan could you please take a look at this again, now it works with dps, and added flag `--dps`, the following is the screenshot at my side and it's ready for merge:
screen shot 2015-02-25 at 12 28 11 pm

Thanks !!!
